lib/timeline/component/item/ClusterItem.js

```diff
@@ -155,18 +152,27 @@ class ClusterItem extends Item {
     }
   
     if (this.options.showStipes) {
+      this.dom.line.style.display = this._isStipeVisible() ? 'block' : 'none';
+      this.dom.dot.style.display = this._isStipeVisible() ? 'block' : 'none';
+
       if (this._isStipeVisible()) {
-        this.dom.line.style.display = 'block';
-        this.dom.dot.style.display = 'block';
-  
-        this.dom.line.style.left = this.dom.line.offsetWidth + start + (end - start) / 2 + 'px';
-        this.dom.dot.style.left = this.dom.line.offsetWidth + start + (end - start) / 2 - this.dom.dot.offsetWidth / 2 + 'px'
-      } else {
-        this.dom.line.style.display = 'none';
-        this.dom.dot.style.display = 'none';
+        this.repositionStype(start, end);
       }
     }
   }
+
+  repositionStype(start, end) {
```

Comment:
What's 'stype'?

Reply:
'stype' is a line with dot to time axes.